apps/native/test/main.c

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *******************************************************************************/
+
+/* Included Files. */
+#include <stdio.h>
+#include <orchestration.h>
+#include <pthread.h>
+#include <unistd.h>
+
+/* Private Data. */
+/* All static data definitions appear here. */
+RequestServiceInfo rsi = { 
+  .ExecutionType  = "native",
```

Comment:
```suggestion
    .ExecutionType  = "native",
```

Reply:
@MoonkiHong @suresh-lc 
1. Regarding the structure of the code, I like the samples/native/test you suggested. Write the final version and I'll make the changes.
2. I use two spaces for indentation. If you think that need 4 - I will do it. Write me please.

P.S. Do we have a description of the code style?

Reply:
@tdrozdovsky Please refer to my comment as follows:
1. Yes, we will look forward to your changed PR. Thank you. ^^
2. Please use the 4 spaces for indentation.

When it comes to the code style, we need to put it into the current developer guideline document as soon as possible. Thank you for your pointing.

Reply:
How about `samples/native/test` folder name?

Reply:
@tdrozdovsky LGTM.